### PR TITLE
Let users select which color to use as the base

### PR DESCRIPTION
**Pull request by CharlieS1103**

### CODE REVIEW

**Comment by CharlieS1103** [comment]
@theRealPadster Well the underscores is just so the settings dropdown can all have the same styling as the other one while still not allowing for spaces. And I think that this amount of customization would be good to allow, as each of the main values allows to get a different part of the album arts color (prominent will get the most common color in the image, vibrant will get the brightest, and vibrant non alarming will get a muted version of the brightest color). Then the pallette generation dictates whether the main color will be the base and the grays and such will be the bg or the inverse, along with all the analog and triad and quad shenanigans.

**Comment by theRealPadster** [comment]
I think it's too much for most people, and they won't know what different options do. I would mess around with every single one and try and find what they do or what looks good and it's just a lot. Might help to have a description for each if we leave the option in. And I think we should maybe take out the triad/quad/etc options out since from my experience they mostly look hideous.

**Comment by CharlieS1103** [comment]
Believe it's ready now

**Review by kyrie25 — DISMISSED** [review]

File: src/components/Modals/Settings/ConfigRow.tsx [hunk 1]
```diff
@@ -36,21 +38,36 @@ const ConfigRow = (props: {
...
+          }}
+        />
+
+        {/*eslint-disable-next-line react/no-children-prop, react/jsx-no-undef*/}
+        <Spicetify.ReactComponent.TooltipWrapper children={<div><TooltipIcon></TooltipIcon></div>}  label={
+          <div>
+            {props.description.split("\n").map(line => {
+              return <>{line}<br /></>;
+            })}
+          </div>} renderInline={true} showDelay={10} placement="top" labelClassName="marketplace-settings-tooltip" disabled={false}></Spicetify.ReactComponent.TooltipWrapper>
```
[the remark this answers is not in the document]
Comment (kyrie25) [reply src/components/Modals/Settings/ConfigRow.tsx]: Tooltip wrapper is meant to be used on one element as children and accepts a string as label, so the tooltip would display when hovering over that element
What stopped you from writing conventional JSX

[the remark this answers is not in the document]
Comment (theRealPadster) [reply src/components/Modals/Settings/ConfigRow.tsx]: Should be better now

**Comment by theRealPadster** [comment]
Can we make these human readable and not camel case? 
image

**Comment by CharlieS1103** [comment]
> Can we make these human readable and not camel case? image

This would be surprisingly annoying to do, I can do it if you'd like but it'll have to be tomorrow or whenever I get a random burst of motivation

**Review by theRealPadster — APPROVED** [review]
Good now?

**Pull request merged**